Jenkinsfile

```diff
@@ -345,8 +345,13 @@ def archiveTestOutput(Map args = [:]) {
     }
     cmd(label: 'Prepare test output', script: 'python .ci/scripts/pre_archive_test.py')
     dir('build') {
+      if (isUnix()) {
+        cmd(label: 'Delete folders that are causing exceptions (See JENKINS-58421)',
+            returnStatus: true,
+            script: 'rm -rf ve || true; find . -type d -name vendor -exec rm -r {} \\;')
```

Comment:
I've no clue where the `ve` folder is defined, but it's there for the `libbeat`

```
[2020-10-05T11:59:25.997Z] + echo libbeat-build
[2020-10-05T11:59:25.998Z] libbeat-build
[2020-10-05T11:59:25.998Z] + ls -ltrah
[2020-10-05T11:59:25.998Z] total 20K
[2020-10-05T11:59:25.998Z] drwxr-xr-x 258 jenkins jenkins 4.0K Oct  5 11:51 docker-gocache
[2020-10-05T11:59:25.998Z] drwxr-xr-x   3 jenkins jenkins 4.0K Oct  5 11:56 ve
[2020-10-05T11:59:25.998Z] drwxr-xr-x   3 jenkins jenkins 4.0K Oct  5 11:59 libbeat
[2020-10-05T11:59:25.998Z] drwxr-xr-x   5 jenkins jenkins 4.0K Oct  5 11:59 .
[2020-10-05T11:59:25.998Z] drwxr-xr-x  26 jenkins jenkins 4.0K Oct  5 11:59 ..
[2020-10-05T11:59:25.998Z] + find . -name build -ls
[2020-10-05T11:59:25.998Z]   5167709      4 drwxr-xr-x   3 jenkins  jenkins      4096 Oct  5 11:59 ./libbeat/build
[2020-10-05T11:59:25.998Z]   5430897      4 drwxr-xr-x   3 jenkins  jenkins      4096 Oct  5 11:57 ./ve/docker/lib/python3.7/site-packages/pip/_internal/operations/build
``

Reply:
IIRC it's created for the integration tests and is created by a Makefile target. It's a Python virtual environment. Should be safe to delete.